application.go

```diff
@@ -36,23 +37,23 @@ type Applications struct {
 // Application is the definition for an application in marathon
 type Application struct {
 	ID                    string              `json:"id,omitempty"`
-	Cmd                   string              `json:"cmd,omitempty"`
-	Args                  []string            `json:"args"`
-	Constraints           [][]string          `json:"constraints"`
+	Cmd                   *string             `json:"cmd,omitempty"`
+	Args                  *[]string           `json:"args,omitempty"`
```

Comment:
no, but I tested to leave `Cmd` blank without `Args` being set and marathon did not complain. Depending on the executor, this could be valid behavior I guess. For the one I used, marathon executed `/bin/sh ""` which did not make a lot of sense -- however: the executor could be a different one I guess?